src/ort_genai.h

```diff
@@ -59,6 +61,44 @@ inline void OgaCheckResult(OgaResult* result) {
   }
 }
 
+struct OgaLoraManager {
```

Comment:
Can you please add a code snippet in the PR to show how this feature will be used?

Reply:
This API no longer exists.